cli/tests/pcluster/validators/test_ec2_validators.py

```diff
@@ -332,18 +332,38 @@ def test_instance_type_base_ami_compatible_validator(
     assert_failure_messages(actual_failures, expected_message)
 
 
+GoodKeyPairsDict = {"KeyPairs": [{"KeyType": "ed25519"}]}
+BadKeyPairsDict = {"KeyPairs": [{"KeyType": "rsa"}]}
```

Comment:
*Minor*: variable names are a bit misleading because strictly speaking there are not good and bad keypairs: RSA is a good keypair for AL2, right?